drivers/hmis/app/graphql/types/hmis_schema/file.rb

```diff
@@ -17,7 +17,7 @@ class HmisSchema::File < Types::BaseObject
     field :expiration_date, GraphQL::Types::ISO8601Date, null: true
     field :confidential, Boolean, null: true
     field :updated_at, GraphQL::Types::ISO8601DateTime, null: false
-    field :updated_by, Types::HmisSchema::User, null: false
+    field :updated_by, Types::Application::User, null: false
```

Comment:
1. The model and database specify that this is nullable, so we need to match that here
2. Can you add a `field :uploaded_by, Types::Application::User, null: true` while we're at it?

Reply:
@ZachCMP these should be `null: true` - did you forget to push a commit?

Reply:
I'm getting errors like `Cannot return null for non-nullable field File.updatedBy` which we don't want